### ISSUE TITLE
Allow me to correct my entries

### ISSUE THREAD

**Issue by bkuker** [cgonyeo/brandrethstats#18]
I have shitty handwriting and a lot of "?" in my entries. Once I can see the image of the original page I could decipher my bullshit and suggest a correction.

**Comment by cgonyeo** [comment]
Good idea, I can provide you with access to the pictures. Also, don't you currently have physical access to the guest books?
